packages/cyberstorm/src/components/Tooltip.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { PropsWithChildren } from "react";
+import * as RadixTooltip from "@radix-ui/react-tooltip";
+import styles from "./componentStyles/Tooltip.module.css";
+
+export interface TooltipProps {
+  label?: string;
+  tooltipStyle?: "tooltip__default";
+}
+
+// Cyberstorm Tooltip component
+
+export const Tooltip: React.FC<PropsWithChildren<TooltipProps>> = ({
+  label,
+  tooltipStyle,
+  children,
+}) => {
+  // const { label, tooltipStyle } = props;
+  return (
+    <RadixTooltip.Provider delayDuration={150}>
+      <RadixTooltip.Root>
+        <RadixTooltip.Trigger>{children}</RadixTooltip.Trigger>
+        <RadixTooltip.Portal>
+          <RadixTooltip.Content
+            /* TS is not aware of defaultProps of function components. */
+            /* eslint-disable-next-line @typescript-eslint/no-non-null-assertion */
+            className={`${styles.tooltip} ${styles[tooltipStyle!]}`}
```

Comment:
As discussed in Discord, this and other similar usages have to be changed. A side benefit is that you can get rid of the eslint disabling comments.